sdk/monitor/azure-monitor-opentelemetry/tests/autoinstrumentation/test_configurator.py

```diff
@@ -13,33 +13,85 @@
 
 
 class TestConfigurator(TestCase):
+    @patch.dict("os.environ", {}, clear=True)
```

Comment:
You can add os.environ.clear() to `setUp()` function to run before every test.

Reply:
Some tests have env vars though. Seems superfluous to redefine the mock then. But I can do that if you feel that overlap is better.

Reply:
Done with class level patch